app/models/tutorial.rb

```diff
@@ -4,4 +4,8 @@ class Tutorial < ActiveRecord::Base
   pg_search_scope :search_by_title, :against => :title
   
   mount_uploader :video, VideoUploader
+
+  def get_all_tutorials_for_company(user_ids_arr)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)